#!/bin/sh /usr/share/dpatch/dpatch-run

## 02_skinelchi-1.3.32.dpatch by Thomas Günther <tom@toms-cafe.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changes for VDR >= 1.3.32.

@DPATCH@
--- skinelchi-0.0.1j/DisplayChannel.h
+++ skinelchi-0.0.1j/DisplayChannel.h
@@ -2,6 +2,7 @@
 #define __DISPLAYCHANNEL_H
 
 #include <vdr/skins.h>
+#include <vdr/device.h>
 #include "setup.h"
 
 class cSkinElchiDisplayChannel : public cSkinDisplayChannel {
